infra/terraform/modules/control_panel_api/role.tf

```diff
@@ -40,7 +40,9 @@ resource "aws_iam_policy" "control_panel_api" {
         "s3:CreateBucket",
         "s3:PutBucketLogging",
         "s3:PutBucketPublicAccessBlock",
-        "s3:PutEncryptionConfiguration"
+        "s3:PutEncryptionConfiguration",
+        "s3:PutBucketVersioning",
+        "s3:PutLifecycleConfiguration"
```

Comment:
This should give the CP enough permissions but you never know 😬

Reply:
oh, add a trailing comma if you wish (it should be allowed in Terraform right?), it's not really a big deal but it helps when adding new things and with the commit history, etc...

Reply:
Normally I would... but I'm following the trailing comma convention I found already in that file... ;-)